CHANGELOG.md

```diff
@@ -14,6 +14,21 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+
+## [2.50.1] - 2022-06-15
```

Comment:
TODO: update date once PR is reviewed/ready for merge

Reply:
[8372a09](https://github.com/cognitedata/cognite-sdk-python/pull/928/commits/8372a0940ee145a0239706e4e673d1cdb3b0688a)